wfe/wfe.go

```diff
@@ -453,6 +454,11 @@ func (wfe *WebFrontEndImpl) verifyPOST(ctx context.Context, logEvent *requestEve
 		logEvent.Contacts = reg.Contact
 	}
 
+	if wfe.AllowAccountDeactivation && reg.Status != core.StatusValid {
+		fmt.Println(reg)
+		return nil, nil, reg, probs.Unauthorized("Cannot use a non-valid registration")
```

Comment:
Nit: this message might be clearer if it was "Cannot deactivate a non-valid registration".